airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -69,3 +79,52 @@ def get_assets(
         assets=[AssetResponse.model_validate(asset, from_attributes=True) for asset in assets],
         total_entries=total_entries,
     )
+
+
+@assets_router.get(
+    "/events",
+    responses=create_openapi_http_exception_doc([401, 403, 404]),
+)
+async def get_asset_events(
```

Comment:
The definition of routes have changed recently in https://github.com/apache/airflow/pull/43797

Route should be decalred `sync` 
```suggestion
def get_asset_events(
```

Reply:
Done